requirements/requirements.txt

```diff
@@ -7,4 +7,4 @@ git+https://github.com/manelmontilla/altimeter.git@dev
 # neptune_python_utils dependencies.
 boto3==1.20.44
 backoff==1.11.1
-aiohttp==3.8.1
+aiohttp==3.8.
```

Comment:
Can we pin an exact version? We should create another PR to start using pip-compile in this repo.

Reply:
Absolutely, I'm pinning it and also the resolving the conflicts.